File: src/main/java/service/AgencyService.java
package service;

import domain.Agency;
import repository.AgencyRepo;

import java.util.Collection;

public class AgencyService {
    private AgencyRepo repo;

    public AgencyService(AgencyRepo repo) {
        this.repo = repo;
    }

    public Collection<Agency> getAllAgencies() {
        return repo.getAll();
    }

    public Agency getByName(String name) { return repo.getByName(name); }
}

File: src/service/AgencyService.cs
using System;
using System.Collections.Generic;
using AgentiiDeTurism.src.domain;
using AgentiiDeTurism.src.repository;

namespace AgentiiDeTurism.src.service
{
    public class AgencyService
    {
        private IAgencyRepo repo;
        public AgencyService(IAgencyRepo repo)
        {
            this.repo = repo;
        }
        public ICollection<Agency> getAllAgencies()
        {
            return repo.getAll();
        }

        public Agency getByName(String name) { return repo.getByName(name); }
    }
}